src/Analyser/Header/Useragent/Browser.js

```diff
@@ -248,17 +254,17 @@ class Browser {
 
         /* Oculus Chromium based browsers */
         if ((match = /OculusBrowser\/([0-9.]*)/u.exec(ua))) {
-          this.data.browser.name = 'Oculus Internet';
+          this.data.browser.name = 'Oculus Browser';
           this.data.browser.channel = null;
           this.data.browser.stock = true;
           this.data.browser.version = new Version({
             value: match[1],
             details: 2,
           });
 
-          if (/Mobile VR/.test(ua)) {
-            this.data.device.manufacturer = 'Samsung';
-            this.data.device.model = 'Gear VR';
+          if (/Pacific/.test(ua)) {
```

Comment:
Can you add this regex test after the Mobile VR one without replacing it?

Reply:
Agreed. 

It will also work properly without the /Mobile VR/ regex. Because the Oculus user agent string contains the SamsungBrowser token. WhichBrowser will first detect Samsung Internet on the Gear VR and then overwrite the browser with Oculus Browser.

However, with the PHP version of WhichBrowser I've also included both regex, to make the intention more clear and not rely on Samsung Internet detection: https://github.com/WhichBrowser/Parser-PHP/commit/87a27c504ab90494a692e640130ec917c54e94bd